lib/gcloud/vision.rb

```diff
@@ -41,7 +41,11 @@ module Gcloud
   #
   #   gcloud = Gcloud.new
   #   vision = gcloud.vision
-  #   # ...
+  #
+  #   image = vision.image "./acceptance/data/landmark.jpg"
```

Comment:
All the code examples are using the images files in the acceptance directory. This directory is not included in the released gem, and these paths won't resolve. Because of this I think these paths should use `/path/to/<image>` instead.

We use `/path/to` in the [Storage code examples](https://quartzmo.github.io/gcloud-ruby/#/docs/master/gcloud/storage) as well.